src/Backend/Core/Ajax/GenerateUrl.php

```diff
@@ -22,10 +22,10 @@ public function execute(): void
         parent::execute();
 
         // get parameters
-        $url = \SpoonFilter::getPostValue('url', null, '', 'string');
-        $className = \SpoonFilter::getPostValue('className', null, '', 'string');
-        $methodName = \SpoonFilter::getPostValue('methodName', null, '', 'string');
-        $parameters = \SpoonFilter::getPostValue('parameters', null, '', 'string');
+        $url = $this->getRequest()->request->get('url', '');
```

Comment:
Would be better if the code uses the default `null`.
`$this->getRequest()->request->get('url');`
=> Maybe change further code in this method?